### DISCUSSION TITLE
梦璃 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#138] [模板讨论]
梦璃 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by windyhun** [comment]


**Reply by wjf0214** [reply]
更新了。

**Comment by windyhun** [comment]
....又又又不行了？
日志：Failed at 1/6 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 403: Forbidden,

**Comment by witherleaves** [comment]
最新的登录好像多了个验证码，好像不能用密码账号登陆了吧

**Comment by miaoxiaozhao** [comment]
Failed at 1/6 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 403: Forbidden,
Request URL: https://moeli-desu.com/wp-admin/admin-ajax.php?action=5660885f3d979a4a6a47d598b9fa9620&9f75f28f6a51eacc6bb29478679b56bd%5Btype%5D=checkUnread&b7e10788857c53d4fc49515bbb1f41e%5Btype%5D=getHomepagePosts
